text/0000-LogService-update.md

```diff
@@ -0,0 +1,189 @@
+- Start Date: 2024-05-15
+- RFC PR: (in a subsequent commit to the PR, fill me with the PR's URL)
+- CppMicroServices Issue: 
+
+# Update LogService class in CppMicroServices
+
+## Summary
+
+The Log Service provides a general purpose message logger for the OSGi framework. It consists of several services: a service for obtaining Loggers to log information and other services for retrieving current or previously recorded log information.
+This task is to support latest OSGi features in LogService class (according to OSGi spec 8- https://docs.osgi.org/specification/osgi.core/8.0.0/service.log.html)
+
+## Motivation
+
+The current CppMicroServices LogService API and implementation that we ship with CppMicroServices does not confirm to the latest LogService OSGi spec.
+The current 'LogService' class diagram looks like-
+
+![Old LogService class](https://github.com/architapruthi/rfcs/blob/master/logService_old.png)
+
+Everything within red is not supported.
+
+Now, according to the latest OSGi spec, the 'LogService' looks like-
+
+![New LogService class](https://github.com/architapruthi/rfcs/blob/master/logService_new.png)
```

Comment:
please add these two pictures to this PR and modify the links so the images aren't linked to https://github.com/architapruthi/rfcs